lang/java/reef-common/src/main/java/org/apache/reef/driver/restart/DriverRestartManager.java

```diff
@@ -40,16 +38,14 @@
 public final class DriverRestartManager {
```

Comment:
There are some methods in this class which are not used (any more?):

  * `setEvaluatorExpired`
  * `setEvaluatorReregistered`
  * `isRestartCompleted`
  * `hasRestarted`

Will they be used in future commits?